app/javascript/packs/puerto-rico-guidance.ts

```diff
@@ -0,0 +1,18 @@
+function showOrHidePuertoRicoExtras(forStateCode) {
+  const isPuertoRico = forStateCode === 'PR';
+
+  elements = document.getElementsByClassName('puerto-rico-extras');
+  for (var i = 0; i < elements.length; ++i) {
+    elements.item(i)?.classList?.toggle('display-none', !isPuertoRico);
+  }
```

Comment:
Using `querySelectorAll` + `forEach` might let you avoid some of the optional chaining here.

```suggestion
  document.querySelectorAll('.puerto-rico-extras').forEach((element) => {
    element.classList.toggle('display-none', !isPuertoRico);
  });
```

Reply:
Changed